src/main.c

```diff
@@ -173,6 +217,63 @@ void WriteInterfaces()
     free(InterfaceArray);
 }
 
+inline int
+CombineMetadataWithPacket(
+    _In_ HANDLE File,
+    _In_ PCHAR FragBuf,
+    _In_ unsigned long FragLength,
+    _In_ long InterfaceId,
+    _In_ long IsSend,
+    _In_ long TimeStampHigh, // usec (unless if_tsresol is used)
+    _In_ long TimeStampLow,
+    _In_ PDOT11_EXTSTA_RECV_CONTEXT Metadata,
+    _In_ unsigned long ProcessId
+)
```

Comment:
The existing code style is

type
FuncName(
    type ParamA,
    type ParamB
    )
{
}

Notice the closing paren is indented. Fix throughout PR.

Reply:
Evidently github strips indentation ?! The main thing I was saying here is, indent the closing paren.

Reply:
OK, I just notice that when copying and pasting, VS is removing these spaces automatically. Even on functions I never changed. I am working on another commit, to re-add them. Its definitely frustrating though because this is against what VS wants to do apparently.